### ISSUE TITLE
Do I need to modify charset in config.py in order to recog Chinese?

### ISSUE THREAD

**Issue by ar8327** [wushilian/CRNN_Attention_OCR#41]
It seems to me that in order to have K Chinese characters recoginized , I have to initialize K logits in the compute graph.With the attention mechanism,there will be K^2 attention nodes in the graph.
Considering there're many Chinese characters(20000+),the compute graph is very large and fails to allocate memory on a dual GTX1080Ti machine (22GB of VRAM)
Is my understanding correct?
